systemtests/bin/download_sym_files.py

```diff
@@ -25,18 +39,92 @@ def emit(self, record):
 METRICS = markus.get_metrics()
 
 
+# Check the response headers for both the Tecken redirect response and the storage
+# backend response.
+def check_headers(response):
+    for item in response.history:
+        if item.status_code in (301, 302):
+            click.echo(f">>> GET redirect: {item.url}")
+            for expected_key, expected_value in TECKEN_RESPONSE_HEADERS.items():
+                if expected_key not in item.headers:
+                    click.echo(
+                        click.style(
+                            f">>> FAIL: GET expected redirect response header {expected_key} is missing.",
+                            fg="red",
+                        )
+                    )
+                    continue
+
+                if item.headers[expected_key] == expected_value:
+                    click.echo(
+                        click.style(
+                            f">>> SUCCESS: GET expected redirect response header {expected_key}: {expected_value}.",
+                            fg="green",
+                        )
+                    )
+                else:
+                    click.echo(
+                        click.style(
+                            f"""
+>>> FAIL: GET expected response header {expected_key} does not have expected value:
+actual: {item.headers[expected_key]}
+expected: {expected_value}
```

Comment:
This is a bit ugly. The motivation is to get `actual` and `expected` on new lines. There's probably a better way to do this in Python.

I know we're going to be moving some of these tests to pytest soon, so maybe it doesn't matter that much in the short term.

Reply:
Yeah, it's fine. We'll let pytest take care of the details here soon.